foundations/foundation-metrics/src/main/java/io/servicecomb/foundation/metrics/Metrics.java

```diff
@@ -1,80 +0,0 @@
-/*
```

Comment:
we should notify liubao, this will make perfStat handler not work

Reply:
Had Recovery,we may keep it for a long time